.circleci/config.yml

```diff
@@ -251,6 +267,20 @@ jobs:
                   - /home/circleci/.cache/torch/
                   - /home/circleci/.cache/gpt4all/
 
+    # Collect the testmondata only on the staging branch 
+      - when:
+          condition:
+            and:
+              - equal: [ staging, << pipeline.git.branch >> ]
+              - equal: [ LONG INTEGRATION, << parameters.mode >> ]
+              - equal: [ "3.10", << parameters.v >> ]
+              - equal: [ DISABLED, << parameters.ray >> ] 
+          steps:
+          - save_cache:
```

Comment:
Not clear why we don't need a indentation here. But circle ci complains when we have an indentation here.